contents/handbook/small-teams/marketing/mission.mdx

```diff
@@ -1 +1 @@
-Increase the number of [ICP](/handbook/strategy/ideal-customer-persona) visitors to our website
+Make PostHog a hub and product engineers to learn and collaborate
```

Comment:
```suggestion
Make PostHog a hub for product engineers to learn and collaborate
```

Reply:
so many writers and James was the one to catch this